ofctrl/multipart_test.go

```diff
@@ -0,0 +1,76 @@
+// #nosec G404: random number generator not used for security purposes
+package ofctrl
+
+import (
+	"fmt"
+	"sync"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"antrea.io/libOpenflow/openflow13"
+)
+
+type multipartActor struct {
+	*OfActor
+	expectedTxs map[uint32]chan *openflow13.MultipartReply
+}
+
+func (o *multipartActor) MultipartReply(sw *OFSwitch, rep *openflow13.MultipartReply) {
+	if ch, found := o.expectedTxs[rep.Xid]; found {
+		ch <- rep
+	}
+}
+func TestMultipartReplyOrder(t *testing.T) {
```

Comment:
I think you should explain why the test is named like this with a comment, or rename the test to `TestMultipartReply` or `TestMultipartReplyTableFeatures`

Reply:
Updated